test/features/showandhide/collectionOfComplex.feature

```diff
@@ -2,27 +2,11 @@
 Feature: Set of scenarios testing show & hide functionality using conditions based on fields in collections of complex
 
   Background:
-    Given the definition sheet 'CaseField' looks like this
-      | CaseTypeID               | ID       | FieldType |
-      | ComplexCollectionComplex | MySchool | School    |
-    Given the definition sheet 'ComplexTypes' looks like this
-      | ID          | ListElementCode                 | FieldType       | FieldTypeParameter |
-      | Child       | IsAutistic                      | YesOrNo         |                    |
-      | Child       | NeedsSupport                    | YesOrNo         |                    |
-      | Family      | Children                        | Collection      | Child              |
-      | SchoolClass | ClassMembers                    | Collection      | Family             |
-      | School      | ProvidesAutisticChildrenSupport | YesOrNo         |                    |
-      | School      | Class                           | Collection      | SchoolClass        |
-    Given the definition sheet 'CaseEventToFields' looks like this
-      | CaseTypeID               | CaseEventID  | CaseFieldID | DisplayContext | PageID                    |
-      | ComplexCollectionComplex | createSchool | MySchool    | COMPLEX        | SingleFormPageWithComplex |
-    Given the definition sheet 'EventToComplexTypes' looks like this
-      | ID	        | CaseEventID  | CaseFieldId | ListElementCode                          | FieldShowCondition                                                                                       |
-      | SchoolClass | createSchool | MySchool    | Name                                     |                                                                                                          |
-      | SchoolClass | createSchool | MySchool    | ProvidesAutisticChildrenSupport          |                                                                                                          |
-      | SchoolClass | createSchool | MySchool    | Class.ClassMembers.Children.IsAutistic   | MySchool.ProvidesAutisticChildrenSupport="Yes"                                                           |
-      | SchoolClass | createSchool | MySchool    | Class.ClassMembers.Children.NeedsSupport | MySchool.ProvidesAutisticChildrenSupport="Yes" AND MySchool.Class.ClassMembers.Children.IsAutistic="Yes" |
-
+    Given the following definition for 'case fields'
```

Comment:
Can we rename this step:
`Given the following definition for 'complex case fields'` ?
Just makes it a bit more explicit the fact that these are complex fields. As it is, it's only understandable looking at the presence of the Path column

Reply:
👍